pkg/skaffold/constants/constants.go

```diff
@@ -59,6 +59,8 @@ const (
 	UpdateCheckEnvironmentVariable = "SKAFFOLD_UPDATE_CHECK"
 
 	DefaultCloudBuildDockerImage = "gcr.io/cloud-builders/docker"
+	DefaultCloudBuildMavenImage  = "gcr.io/cloud-builders/mvn"
```

Comment:
WDYT about using the gcr.io/k8s-skaffold/skaffold image for every builder?

Then we wouldn't have to use multiple images

Reply:
Copy the context + skaffold.yaml, then invoke `skaffold build` and you get all the builders for free.

Reply:
I don't know about that. I have a few questions:
 + Is skaffold pre-pulled on GCB?
 + How would you build only one artifact with `skaffold build`?
 + Isn't that a bit too inception-ish?